### PR TITLE
Added HostUp

### PR DESCRIPTION
**Pull request by hostup**
Hello,
we just started accepting bitcoin cash after some requests and now want to get listed here. I hope that's all the changes needing to be done, if not let me know!

### CODE REVIEW

File: _data/hosting.yml [hunk 1]
```diff
@@ -340,6 +340,19 @@ websites:
   bch: true
   btc: true
   othercrypto: true
+  - name: HostUp
```
Comment (EchterAgo) [inline _data/hosting.yml]: This line needs to start at column 0 like the other entries.

**Comment by EchterAgo** [comment]
Are you using BitPay? If so, please add this at the end:

```
  keywords:
  - bitpay
  - bip70
```

**Comment by hostup** [comment]
> Are you using BitPay? If so, please add this at the end:
> 
> ```
>   keywords:
>   - bitpay
>   - bip70
> ```

Yes. We just started accepting bitpay. Before this we have been trying to get blockonomics to accept bitcoin cash for months. It seems as they accept it now, but I am still awaiting full support.

**Comment by EchterAgo** [comment]
image

This is how the listing looks now. The addition will be visible at https://beta.acceptbitco.in/ soon. Deployment to the main site will be next week.

**Pull request merged**